src/Zenaton/v1/Model/Scheduling/Schedule.php

```diff
@@ -0,0 +1,108 @@
+<?php
+
+namespace Zenaton\Model\Scheduling;
+
+final class Schedule
+{
+    const API_DATE_FORMAT = 'Y-m-d\TH:i:s.u\Z';
+
+    /** @var string */
+    private $id;
+    /** @var string */
+    private $cron;
+    /** @var null|string */
+    private $name;
+    /** @var TaskTarget|WorkflowTarget */
+    private $target;
+    /** @var \DateTimeInterface */
+    private $insertedAt;
+    /** @var \DateTimeInterface */
+    private $updatedAt;
+
+    private function __construct()
+    {
+    }
+
+    /**
+     * @internal should not be called by user code
+     *
+     * @return self
+     */
+    public static function fromArray(array $values = [])
+    {
+        $schedule = new static();
+        $schedule->id = (string) $values['id'];
+        $schedule->cron = (string) $values['cron'];
+        $schedule->name = isset($values['name']) ? (string) $values['name'] : null;
+        $schedule->target = static::createTarget($values['target']);
+        $schedule->insertedAt = \DateTimeImmutable::createFromFormat(static::API_DATE_FORMAT, $values['insertedAt']);
```

Comment:
Is this UTC or does it have a timezone to it? @MrYawe maybe knows what the api expects?

Reply:
Dates coming from Alfred are all UTC afaik